x-pack/plugins/observability_solution/slo/server/routes/slo/route.ts

```diff
@@ -141,7 +141,7 @@ const createSLORoute = createSloServerRoute({
 });
 
 const inspectSLORoute = createSloServerRoute({
-  endpoint: 'POST /internal/api/observability/slos/_inspect 2023-10-31',
+  endpoint: 'POST /internal/observability/slos/_inspect 2023-10-31',
```

Comment:
versioned and internal api doesn't go along, not sure what happened there :D

Reply:
I'll 🔪 it